tool/jt.rb

```diff
@@ -232,9 +233,64 @@ def run(*args)
+                 json/test_json_string_matching.rb json/test_json_unicode.rb logger/test_logger.rb matrix/test_matrix.rb matrix/test_vector.rb monitor/test_monitor.rb
+                 net/ftp/test_ftp.rb net/http/test_buffered_io.rb net/http/test_http.rb net/http/test_http_request.rb net/http/test_httpheader.rb net/http/test_httpresponse.rb
+                 net/http/test_httpresponses.rb net/http/test_https.rb net/http/test_https_proxy.rb net/imap/test_imap_response_parser.rb net/pop/test_pop.rb net/protocol/test_protocol.rb
+                 net/smtp/test_response.rb net/smtp/test_smtp.rb net/smtp/test_ssl_socket.rb nkf/test_kconv.rb nkf/test_nkf.rb optparse/test_bash_completion.rb optparse/test_getopts.rb
+                 optparse/test_noarg.rb optparse/test_optarg.rb optparse/test_optparse.rb optparse/test_placearg.rb optparse/test_reqarg.rb optparse/test_summary.rb
+                 optparse/test_zsh_completion.rb ostruct/test_ostruct.rb pathname/test_pathname.rb psych/test_alias_and_anchor.rb psych/test_array.rb psych/test_boolean.rb
+                 psych/test_class.rb psych/test_coder.rb psych/test_date_time.rb psych/test_deprecated.rb psych/test_document.rb psych/test_emitter.rb psych/test_encoding.rb
+                 psych/test_exception.rb psych/test_hash.rb psych/test_json_tree.rb psych/test_merge_keys.rb psych/test_nil.rb psych/test_null.rb psych/test_numeric.rb
+                 psych/test_object.rb psych/test_object_references.rb psych/test_omap.rb psych/test_parser.rb psych/test_psych.rb psych/test_scalar.rb psych/test_scalar_scanner.rb
+                 psych/test_serialize_subclasses.rb psych/test_set.rb psych/test_stream.rb psych/test_string.rb psych/test_struct.rb psych/test_symbol.rb
+                 psych/test_tainted.rb psych/test_to_yaml_properties.rb psych/test_tree_builder.rb psych/test_yaml.rb psych/test_yamlstore.rb psych/json/test_stream.rb
+                 psych/nodes/test_enumerable.rb psych/visitors/test_depth_first.rb psych/visitors/test_emitter.rb psych/visitors/test_to_ruby.rb psych/visitors/test_yaml_tree.rb
+                 ripper/test_files.rb ripper/test_filter.rb ripper/test_parser_events.rb ripper/test_ripper.rb ripper/test_scanner_events.rb scanf/test_scanf.rb
+                 scanf/test_scanfblocks.rb socket/test_basicsocket.rb socket/test_nonblock.rb socket/test_socket.rb stringio/test_stringio.rb strscan/test_stringscanner.rb
+                 thread/test_queue.rb zlib/test_zlib.rb ruby/enc/test_big5.rb ruby/enc/test_cp949.rb ruby/enc/test_emoji.rb ruby/enc/test_euc_jp.rb ruby/enc/test_euc_kr.rb
+                 ruby/enc/test_euc_tw.rb ruby/enc/test_gb18030.rb ruby/enc/test_gbk.rb ruby/enc/test_iso_8859.rb ruby/enc/test_koi8.rb ruby/enc/test_shift_jis.rb ruby/enc/test_utf16.rb
+                 ruby/enc/test_utf32.rb ruby/enc/test_windows_1251.rb]
```

Comment:
Yes what we really want to do is add support for running `jt test ruby/**/test_.rb` or something like that.